src/corehost/src/servicing_index.cpp

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include "trace.h"
+#include "servicing_index.h"
+
+servicing_index_t::servicing_index_t(const arguments_t& args)
+{
+    m_patch_root = args.svc_dir;
+	if (!m_patch_root.empty())
+	{
+		m_index_file.assign(m_patch_root);
+		append_path(m_index_file, _X("dotnet_servicing_index.txt"));
```

Comment:
We should have the name as a constant so that if it gets used in multiple places in future, we have a single place to fix it.